recipe/meta.yaml

```diff
@@ -8,19 +8,30 @@ package:
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 553978badef3bb95af7557679f037a3f01cc167e9991fd9618578d3971cd62f6
+  sha256: 49a8375d72ba27efe7414a9ccc3ad5f969a13838bd089ad985ab836d03ffb8a4
 
 build:
   number: 0
-  # accelerate not yet available on OSX or Windows
-  skip: true  # [py<37 or osx or win]
+  skip: true  # [py<37]
+  # pytorch is not built for python 3.11 on ppc64le
+  skip: true  # [py>310 and ppc64le]
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+    - wheel
+    - pillow
+    - numpy
 
 outputs:
   - name: {{ name }}-base
     script: build_base.sh  # [not win]
     script: build_base.bat  # [win]
     build:
-      skip: true  # [py<37 or osx or win]
+      skip: true  # [py<37]
+      skip: true  # [py>310 and ppc64le]
```

Comment:
An explanation of why we skip py>310 and ppc64le would be nice to add.

Reply:
Could you add `entry_points` for `{{ name }}-base`:
```
entry_points:
  - diffusers-cli=diffusers.commands.diffusers_cli:main
```
See https://github.com/huggingface/diffusers/blob/5e80827369272f4b24af51573466aba24454b068/setup.py#L250C40-L250C91

And then add to the test/commands for all outputs:
`  - diffusers-cli --help`